src/hooks/useOnClickOutside.ts

```diff
@@ -2,11 +2,13 @@ import { onMounted, onUnmounted, Ref } from '@vue/composition-api';
 
 function useOnClickOutside(
   elementRef: Ref<HTMLElement | null>,
+  testRef: Ref<HTMLElement | null>,
```

Comment:
what is test ref and why we need it?

Reply:
Since the popper moved to the body, and the click on it needs to be tracked too